src/document/DocumentAnnotator.scss

```diff
@@ -10,9 +10,23 @@
         pointer-events: none;
     }
 
+    .textLayer {
```

Comment:
Probably would be helpful to add some comments about why the pointer-events is being switched around based on the classes applied for future readers